ami_backup.py

```diff
@@ -46,6 +46,6 @@ def lambda_handler(event, context):
     ec.create_tags(
```

Comment:
This tag adds to all instances and AMIs
Instance tag with `Key` = `DeleteOn` and `Value` = `2017-09-04` can cause misunderstanding.
Let's add different tags: 
to the instance - `AMIDeleteOn`
to the AMI - `DeleteOn`

Reply:
We should turn off instance tagging. Instance tag `DeleteOn` may cause confusion.
![screenshot from 2017-09-14 18-13-27](https://user-images.githubusercontent.com/11299538/30437894-7572d556-9978-11e7-9f9b-e6cd77f7abcf.png)